Sofascore2025/Sofascore2025/EventView.swift

```diff
@@ -0,0 +1,220 @@
+    private let homeTeamLabel = UILabel()
+    private let awayTeamLabel = UILabel()
+    private let numHomeTeamGoalsLabel = UILabel()
+    private let numAwayTeamGoalsLabel = UILabel()
+    private let mainView = UIView()
+    private let mainStackView = UIStackView()
+
+    internal override func addViews() {
+        addSubview(mainStackView)
+        mainStackView.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+        
+        mainView.addSubview(homeTeamImageView)
+        mainView.addSubview(leftLabel)
+        mainView.addSubview(awayTeamImageView)
+        mainView.addSubview(timeLabel)
+        mainView.addSubview(minuteLabel)
+        mainView.addSubview(homeTeamLabel)
+        mainView.addSubview(awayTeamLabel)
+        mainView.addSubview(numHomeTeamGoalsLabel)
+        mainView.addSubview(numAwayTeamGoalsLabel)
+        
+        mainStackView.addArrangedSubview(mainView)
+
+        styleViews()
+    }
+
+    internal override func styleViews() {
+        homeTeamImageView.contentMode = .scaleAspectFit
+        homeTeamImageView.clipsToBounds = true
+
+        awayTeamImageView.contentMode = .scaleAspectFit
+        awayTeamImageView.clipsToBounds = true
+
+        leftLabel.textAlignment = .center
+        leftLabel.font = UIFont.systemFont(ofSize: 16, weight: .bold)
+        leftLabel.textColor = .black
+
+        timeLabel.textAlignment = .center
+        timeLabel.textColor = .black
+        timeLabel.font = UIFont(name: "Roboto-Regular", size: 12)
+        timeLabel.textColor = UIColor(red: 18/255, green: 18/255, blue: 18/255, alpha: 0.4)
+
+        minuteLabel.textAlignment = .center
+        minuteLabel.font = UIFont(name: "Roboto-Regular", size: 12)
+        minuteLabel.textColor = .black
+        minuteLabel.textColor = UIColor(red: 18/255, green: 18/255, blue: 18/255, alpha: 0.4)
+
+        homeTeamLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        homeTeamLabel.textAlignment = .left
+
+        awayTeamLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        awayTeamLabel.textAlignment = .left
+
+        numHomeTeamGoalsLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        numHomeTeamGoalsLabel.textAlignment = .right
+
+        numAwayTeamGoalsLabel.font = UIFont(name: "Roboto-Regular", size: 14)
+        numAwayTeamGoalsLabel.textAlignment = .right
+
+        mainStackView.axis = .vertical
+        mainStackView.spacing = 0
+        mainStackView.alignment = .fill
+
+        setupConstraints()
+    }
+
+    internal override func setupConstraints() {
+        timeLabel.snp.makeConstraints { make in
+            make.centerX.equalTo(leftLabel)
+            make.width.equalTo(56)
+            make.top.equalTo(leftLabel).offset(10)
+        }
+
+        minuteLabel.snp.makeConstraints { make in
+            make.centerX.equalTo(leftLabel)
+            make.top.equalTo(timeLabel.snp.bottom).offset(4)
+            make.width.equalTo(56)
+        }
```

Comment:
Za sve snp constrainte mozes ovako pisat bit ce ti lakse
```suggestion
        minuteLabel.snp.makeConstraints { 
            $0.centerX.equalTo(leftLabel)
            $0.top.equalTo(timeLabel.snp.bottom).offset(4)
            $0.width.equalTo(56)
        }
```